chat-commands.js

```diff
@@ -1052,6 +1052,7 @@ exports.commands = {
 			this.sendReply("/roompromote - This room isn't designed for per-room moderation");
 			return this.sendReply("Before setting room mods, you need to set it up with /roomowner");
 		}
+		if (!this.canTalk() && !user.can('bypassall')) return;
```

Comment:
You don't need to do `bypassall` separately; I think there's a `bypassall` check in `canTalk` (and if there isn't, it should be added there)